modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -3418,7 +3418,9 @@ public final Bounds getBoundsInParent() {
      * this variable. For example, the x or y variables of a shape, or
      * {@code translateX}, {@code translateY} should never be bound to
      * {@code boundsInParent} for the purpose of positioning the node.
-     * @return the boundsInParent for this {@code Node}
+     *
+     * @return the {@code boundsInParent} property for this {@code Node}
+     * @see {@linkplain Node Bounding Rectangles} section in the class docs
```

Comment:
This causes a javadoc error:

```
> Task :javadoc
modules\javafx.graphics\src\main\java\javafx\scene\Node.java:3423: error: unexpected content
     * @see {@linkplain Node Bounding Rectangles} section in the class docs
       ^
1 error

> Task :javadoc FAILED
```

Reply:
`@see` doesn't show for properties because of the special handling (same goes for `@return`). I opted to create HTML links to the headers instead. I did it to all for them even if they are not all used.